src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -121,3 +122,22 @@ def create(self, bucket, uid):
             cherrypy.response.headers['Content-Type'] = 'application/json'
             cherrypy.response.status = 500
             return {'detail': str(e)}
+
+
+@ApiController('rgw/user')
+@AuthRequired()
+class RgwUser(RESTController):
+
+    def delete(self, uid):
+        try:
+            rgw_client = RgwClient.admin_instance()
+
+            # Ensure the user is not configured to access the Object Gateway.
+            if rgw_client._SYSTEM_USERID == uid:
+                raise DashboardException(msg='Can not delete the user "{}", he is '
```

Comment:
Slight textual improvement suggestion: "Unable to delete {} - this user account is required for managing the Object Gateway"

Reply:
Done